Examples/UIExplorer/UIExplorerIntegrationTests/UIExplorerIntegrationTests.m

```diff
@@ -36,7 +38,8 @@ - (void)setUp
 #endif
 
   NSOperatingSystemVersion version = [NSProcessInfo processInfo].operatingSystemVersion;
-  RCTAssert(version.majorVersion == 8 || version.minorVersion >= 3, @"Tests should be run on iOS 8.3+, found %zd.%zd.%zd", version.majorVersion, version.minorVersion, version.patchVersion);
+  float versionToCompare = [[NSNumber numberWithFloat:version.majorVersion + version.minorVersion / 10.0] floatValue];
+  RCTAssert(versionToCompare >= MIN_VERSION, @"Tests should be run on iOS 8.3+, found %zd.%zd.%zd", version.majorVersion, version.minorVersion, version.patchVersion);
```

Comment:
Versions are not like normal numbers. For example 9.10 is a later version than 9.9. Also need to handle 9.9.1 for example.